package.json

```diff
@@ -26,9 +27,14 @@
   "icon": {
     "64": "webextension/icons/pulse-64.png"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/pulse.git"
+  },
   "scripts": {
-    "build": "./node_modules/webpack/bin/webpack.js",
-    "watch": "./node_modules/webpack/bin/webpack.js --watch",
-    "prettier": "./node_modules/prettier/bin/prettier.js --write src/**/*.js webpack.config.js"
+    "build": "webpack",
+    "watch": "webpack",
+    "prettier": "prettier --single-quote --write src/**/*.js webpack.config.js",
```

Comment:
I also saw this the other day, which may [or may not] be useful: [**prettier-eslint**](https://www.npmjs.com/package/prettier-eslint):

> This formats your code via `prettier`, and then passes the result of that to `eslint --fix`. This way you can get the benefits of prettier's superior formatting capabilities, but also benefit from the configuration capabilities of eslint.